src/mxarray.jl

```diff
@@ -406,8 +406,13 @@ end
 # char arrays and string
 
 function mxarray(s::String)
-    pm = ccall(_mx_create_string, Ptr{Void}, (Ptr{UInt8},), s)
-    MxArray(pm)
+    utf16string = transcode(UInt16, s)
```

Comment:
will make one liners above shorter (not that this repo is strictly adhering to 98 arbitrary rule)